environments/kubernetes/manifests/thanos-querier-cache-deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: observatorium-querier-cache
+  name: observatorium-querier-cache
+  namespace: observatorium
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: observatorium-querier-cache
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: observatorium-querier-cache
+    spec:
+      containers:
+      - args:
+        - -config.file=/etc/cache-config/observatorium-cache-conf.yaml
+        - -frontend.downstream-url=thanos-querier.observatorium.svc.cluster.local:9090
+        env:
+        - name: NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        image: quay.io/cortexproject/cortex:master-8533a216
+        name: observatorium-querier-cache
+        ports:
+        - containerPort: 9001
+          name: http
+        - containerPort: 9005
+          name: grpc
```

Comment:
why this guy needs gRPC?

Reply:
do we need to expose this?

Reply:
I don't think its needed. Might have been a copy pasta artifact. Removing.

Reply:
Removed in https://github.com/observatorium/configuration/pull/52/commits/ece07ba22b77d5c1d47a56c462121e5776b4bf20